.github/workflows/merge-review.yml

```diff
@@ -9,15 +9,15 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Setup Node.js environment
         uses: actions/setup-node@v2.5.0
         with:
           node-version: 14
 
       - name: NPM Install
-        run: npm ci
+        run: npm ci --ignore-scripts
```

Comment:
Adding the `--ignore-scripts` flag prevents installing git hooks with husky via the prepare script.